### ISSUE TITLE
Add Workflow for using Saved Top-K Logits for Distillation

### ISSUE THREAD

**Issue by HannanJaved** [whittle-org/whittle#262]
**Is your feature request related to a problem? Please describe.**
Saving logits from a teacher model so that we can load it when training a student model instead of calculating it dynamically each time

**Comment by HannanJaved** [comment]
Validation loss curve:
<img width="2528" height="1328" alt="Image" src="https://github.com/user-attachments/assets/200da7b9-ca66-4375-a7a7-606f593276a6" />

Spreadsheet containing more values:
[wandb_export_2025-07-22T15_11_20.504+02_00.csv](https://github.com/user-attachments/files/21367572/wandb_export_2025-07-22T15_11_20.504%2B02_00.csv)
